contrib/initramfs/hooks/zfsunlock

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+PREREQ="dropbear"
+
+prereqs() {
+    echo "$PREREQ"
+}
+
+case "$1" in
+    prereqs)
+        prereqs
+        exit 0
+    ;;
+esac
+
+. /usr/share/initramfs-tools/hook-functions
+
+if [ -r "${DESTDIR}/sbin/dropbear" ]; then
+	cat >"$DESTDIR"/bin/zfsunlock <<'EOF'
```

Comment:
I'd still like to see this as a separate file. It'd be easier to work with. For example, I could run `shellcheck` on it trivially.

On that note, can you run shellcheck on your various scripts and address any non-bogus issues it finds?